File: Signal.CPP/IntroFunctionList.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IntroFunctionList.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TIntroFunList *IntroFunList;
//---------------------------------------------------------------------------
__fastcall TIntroFunList::TIntroFunList(TComponent* Owner)
	: TBasicForm3(Owner)
{
	RichEdit1->Lines->LoadFromFile(ExtractFilePath(Application->ExeName)+"\\Routines.rtf");
}
//---------------------------------------------------------------------------

File: Signal.Delphi/IntroFunctionList.pas
unit IntroFunctionList;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  Basic3, StdCtrls, ComCtrls;

type
  TIntroFunList = class(TBasicForm3)
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  IntroFunList: TIntroFunList;

implementation

{$IFDEF CLR}
{$R *.nfm}
{$ELSE}
{$R *.dfm}
{$ENDIF}

procedure TIntroFunList.FormCreate(Sender: TObject);
begin
  inherited;
  With RichEdit1.Lines, RichEdit1 do
  begin
    Clear;
    LoadFromFile(ExtractFilePath(Application.ExeName)+'Routines.rtf');
  end;
end;

initialization
   RegisterClass(TIntroFunList);

end.
